web-ui/src/main/resources/catalog/style/gn_search.less

```diff
@@ -510,3 +510,9 @@ div[gn-transfer-ownership] .panel-body {
     }
   }
 }
+
+.gn-required > label:after {
```

Comment:
I add this to enhance mandatory fields in the WPS process form.
The pull request is here now: #980